### ISSUE TITLE
Callable/dynamic parameters in ss.lognorm_ex

### ISSUE THREAD

**Issue by pausz** [starsimhub/starsim#532]
With `ss.bernoulli()`  is possible to do 

```python
   env2ppl_p_inf=ss.bernoulli(p=self.infection_prob_function)    ## updated later based on function
```
where `self.infection_prob_function` returns an array of probabilities, whose values depend on other variables and parameters,  each time `env2ppl_p_inf`  is called.

I thought that the following would work

```python
dur_prep2next=ss.lognorm_ex(mean=self.prepatent_duration_mean,
                            stdev=self.prepatent_duration_std,
                            dynamic_pars=True)
```

But it doesn't seem plausible because the method `convert_ex_to_im():`  expects mean and stdev to be a float or array-like. Very likely I'm misusing the `dynamic_pars` kwarg too. 

:question: I'm wondering if this use is supported at all in lognorm_ex (and other distributions too).

**Comment by cliffckerr** [comment]
Hi @pausz , I think this is a duplicate of #519 , and should be fixed by #527

**Comment by cliffckerr** [comment]
Closed by #527
